process_pr.py

```diff
@@ -1115,6 +1159,40 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
                 if m.group(2):
                     code_check_apply_patch = True
 
+        selected_cats = []
+
+        if re.match("^([+]1|approve[d]?|sign|signed)$", first_line, re.I):
+            ctype = "+1"
+            selected_cats = copy.copy(commenter_categories)
+        elif re.match("^([-]1|reject|rejected)$", first_line, re.I):
+            ctype = "-1"
+            selected_cats = copy.copy(commenter_categories)
```

Comment:
@iarspider , why not just use `commenter_categories[:]` instead of `copy` module. `commenter_categories` holds just couple of strings